### PR TITLE
Phase 5 API testing with 'Requests'

### PR DESCRIPTION
**Pull request by xenony**

### CODE REVIEW

File: tests/test_jira_api.py [hunk 1]
```diff
@@ -0,0 +1,91 @@
...
+        assert response.status_code == 201
+
+    @allure.tag('api')
+    @allure.title("Update issue")
+    @pytest.mark.api
+    def test_update_issue(self):
+        response = api.post_issue("OA API Issue002 ", "OlegAverkin", "Medium")
+        issue_id = response.json().get('id')
+        response_update = api.update_issue(issue_id, "OA API Issue002 Updated", "OlegAverkin", "Low")
+        response_update.status_code = 204
```
Comment (archick12) [inline tests/test_jira_api.py]: Are you sure you want to assign the value rather than check the value?

**Review by archick12 — APPROVED** [review]

**Review by archick12 — CHANGES_REQUESTED** [review]
Are you sure you want to assign the value rather than check the value?

File: tests/test_jira_api.py (hunk 1 above)
Comment (xenony) [reply tests/test_jira_api.py]: Fixed.

**Review by archick12 — APPROVED** [review]
Good Job!

**Pull request merged**